.php_cs.dist

```diff
@@ -41,7 +42,7 @@ $rules = [
     '@PHPUnit60Migration:risky' => true,
     '@PHP71Migration' => true,
     '@PHP71Migration:risky' => true,
-    'compact_nullable_typehint' => true,
```

Comment:
Please keep the order as is and only add your new rule

Reply:
Done.